server/init.go

```diff
@@ -19,203 +25,331 @@ import (
 	dbm "github.com/tendermint/tmlibs/db"
 )
 
-// testnetInformation contains the info necessary
-// to setup a testnet including this account and validator.
-type testnetInformation struct {
-	Secret string `json:"secret"`
-
-	ChainID   string                   `json:"chain_id"`
-	Account   string                   `json:"account"`
-	Validator tmtypes.GenesisValidator `json:"validator"`
-	NodeID    p2p.ID                   `json:"node_id"`
+// TODO flag to retrieve genesis file / config file from a URL?
```

Comment:
Possible, but maybe easier to display help information for using `curl` / `wget`?

Reply:
Agree - removing this comment

Reply:
Addressed